docs/pages/templates.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Templates
+description: The quickest way to get rocking and rolling with Remix
+order: 3
+---
+
+# Templates
+
+When using [`create-remix`][create-remix] to generate a new project, you can choose a template to quickly get up and running.
+
+## Basic Template
+
+If you run `create-remix` without providing the `--template` option, you'll get a basic template using the [Remix App Server][remix-app-server].
```

Comment:
Should we call out the TS aspect here?  

```suggestion
If you run `create-remix` without providing the `--template` option, you'll get a basic TypeScript template using the [Remix App Server][remix-app-server].
```

And then we could add a little section:

```
If you are not interested in TypeScript, you can install the simpler Javascript template via:

npx create-remix --template @remix-run/remix/templates/remix-javascript
```

Reply:
Ah, nice catch, I forgot to add this.